.travis.yml

```diff
@@ -13,22 +13,42 @@ env:
+  - secure: tibBlNU4AJ/W25sxznoSP74osucZiqD13Yml4Q7RWTRPuf1dXcV95JCocHn+3uKhYxA0ChBNap8FuQf8AmyEEXLoKDPFCL8JLpSX7JY+10/m6Q1peqLgTqqXY03J01d7dct3EYjZ3fRCin2jWYdaL79H9S2uGXmo+b8d5ZwBM8ZRs7xysaGAzp/nOnTlGAjy2xZe2O+/uPeTr4jxN3s6bsp1ef40QazykpI1VNE695Fqdz++aU/OzQP4kqJBsg5FypPvZWq4L4PMujz4GkBk3sF/oZlOuHfGqbRqm57QsEOWZf87I+bEw1XPB2OfCYC3BDPL1thp/mqwCcPx1l66oS6rS7M2FEfW+XwH9Fsr/9+HeVrpyymRJdZvYVYmB/Sapbwp0ZLffxrtbG2hgDXYv2YOkId4XbgjNBEeZY1yNT7WvUZZnhdBTHSj1oyK/SyokY8y612OroZvXOVxU488DEomvrZn9us4wx4S7jlJsKhGru/vqTtgulxXbaCYgTBbRqT84SnQbWrT7NowuhTohxUlfyDVhgeu+H+5e4KiPODswBDkV7KWnUKQF1TN2VQ4k172w67o4p3+ZJDZs3Pj6ObVSqjATtXsZsfP9he5HDHUHfz6KndDSb470BBPYLmCuN4EqjVJtnblKEMp1ruTRsGtoFUpOSttx3UDoYRLuc0=
+  - secure: xUcSlF5u2XsnxdGmxUGqcbm38HzacWgX7sgmD8/BvxmG0AIt9gvNDNmO7BDewQnepjxo8ieM3mO0veGuNCBGOBOJ0eTSryEq03awEFfu9aL2fjoQRvD0/PSLOIIE/jsDb2EsNwHhCA3nI7XznntZThatR15C3NEdKurv/3uMZoOI+ISFQgmXTEz+evQLoBYSwWIEGNpDG+8NDAou9iHipSwwgG1lZqi1IS3fdi3S9BzuTYb5//qYGBDPJp6ui5OxLMoqT3XeiO+8I1V9oAlO/ygqw+CvAGpg+Uemw0PGTiaRf6C4gbdS/TP3WkAcTCNOboAlN8mgj1ZwfwkGWs2HZJumoGTIsPRkzharhT6kMdiEwpQzylXBYuXlzPlLyaZolzZXdqI2ZbwTlqOGmHNzQgKW+wJb4VMg21auBOsJdXwkYlj/J6/J9673l1AjKCU5OwvCCiZ1sDF4om4Wgn7nRbEdjm+6Xh25mYEGbQFnPztdzQwFCFxobqwVLGzv7NrfNVxJ9ESHqoLcVzDwbs1NjDQ55jK9ccsmHc+cDA+/rgkKvPb5UyRX7L0f5sF+65LcFoAxkZQd4uAUl06w2FhomY6yOkeY53AJfLah6FMczbdqXJJU6WParG5u/PrW6jvohVGZJkTaEHswarmVOMMIPWemTtFvkfy6+eserkIdP/4=
+  #- secure: FHdiFoyZQpOSRD5Ex7E5coZyyKeTPXg0FL2hKwJYszBs2EgMggtI6sX2YhawExOdTQuqLb/WHXqNhna4RlFMLU/EkCjk9tdpd+ni54hUY/jIZTxT+bGMgvRyR4WeuHJfDtpSt9yFL6F7TEX9QhFJ6Q597N4/OurwU/Tb/sdPldWp+y4RkdH7TSr/Q82v9HbW03h0RzgS/5Lhtn4Zvj+qUDijwcC37kAM3RoJrH37KYuftkBo1bsuvYPpouC5CwFCrEJMP3FLB5Av0UJjreb3hR6F6j035X8ZpJuuS1T/w5fHOwsVmpCcT5SMeyEV0emyFkj3oikDO3Aun/Gm8YpTZ7Xvo8vkdUHlujNT6xchkwYLOI2WEwOOsIb1+KIU18Kcs8YZR4/VprbRvuTsY7oxMDnXCBHwz1YKUULrsQ9B/gPPngbsU7AB4uwEtMzGl2XvYr+WScuYEl5JZhxinZZSdqNevlikOV+TeAyctVYONuhOnunmQZjNUR5bxpwnfvqXrUSVOcSajf/4MHTSKUXM9YuC3zNVXed9dxC7xk81W21zzi6Pq6GIVwhshF7QTwaDSkbH9PwBB84han7ORaOkzdZCcxLWiFc9aYHEiXVLEZ5RqPIxCdlcWGe2uweG98j8MtkeqjxJGiH2WJyBez6UG15QDHe0aFZQT6+v3LeIMj4=
+  #- secure: RK0zUvbfpiUYjaOhm9J9mp6hThHKeHE5owwRiAFXeVbPo6ewkzNkPS8ZcssX6l4h6uXOfGb3X1Xbl13rKJDtq0wPsYewi5UjDoAYq8oRsE9GCrqn+EQv6RxgSrO1MKZWBsDOJ8qpbLKPKJm3KFxtzo+9URGYNJmIeHsB19m6Y/YWYgMr9+WLSe1cQcMbywoBsW9Kr81XBAYA32LG0RWkDl5NW3vNmXrsMPI1o1pfRYfs0P0UwCtD6z29Ea9wVRqwdKAivFApfKrZftcVRo63JR5ymSbH1IQkoUv4jRlSYRx4eEMxDfySAZVfxkpCOwN55t7YYlqTtTIjlttMuiENXoyCtK8zR7q56mRShPtUSIsWt3dUzDHm1dIrGsei4cEebPymq5aWRDPAFzkUdj6cnoczfV21odHkyH5ryOYyVkbtRcfFvmj9EFIxPjUJOdq7CSLpk1lKMJYW6b6wwE9dU9tFUGX8OcaDevOJQHsDadRikkVB2NcY94/zP9uxkjgFePK4HtnNt56cJ2tKCvPYTVgj1IzTO8ZOy9y5C3IieGHCTK3b1a8LIp9eais9yxnVE66Gqu0ka4SD3AMzBJrdUERjLDfp9LtX5E/x8dz8+gQEI26QBMHAs7Wncz97dlGHXNw6zjZW9IOXrhld8wML+Tzrozcg8KJW00D8bQqlKHU=
 before_install:
 - pip install --user awscli
 install:
 - git clone https://github.com/aws-robotics/travis-scripts.git .ros_ci
 script:
-- . .ros_ci/add_tag.sh
+- ". .ros_ci/add_tag.sh"
```

Comment:
Is this a change we should make to all SA travis.ymls ?

Reply:
It's a linter change performed automatically by Travis when you add a secret.